docs/configuration/options.md

```diff
@@ -421,6 +419,8 @@ Setting this option to _false_ in production is a security risk and may allow se
 
 #### Description
 
+Cookies in NextAuth.js are chunked by default, meaning that once they reach the 4kb limit, we will create a new cookie with the `.{number}` suffix and reassemble the cookies in the correct order when parsing / reading them. This was introduced to avoid size constraints which can occur when users want to store additional custom fields in their sessionToken, for example, or enable encryption, which adds size overhead.
```

Comment:
encryption is kn by default now, user doesn't need to do anything to opt in. options have been simplified i  https://github.com/nextauthjs/next-auth/pull/3062 and that should be reflected in the migration guide. I'm ok with it if that will be a different PR.